src/api/route-services/samples.js

```diff
@@ -29,14 +29,17 @@ class SamplesService {
 
     const response = await dynamodb.query(params).promise();
 
-    if (response.Item) {
-      const prettyResponse = convertToJsObject(response.Item);
+    const items = response.Items ? response.Items[0] : response.Item;
```

Comment:
Sometimes the data is under `Item` - for example in unit testing mockDynamoQuery returns Item (samples.test.js:35)

Reply:
`Sometimes the data is under Item` -- are there any other cases when data is under Item?

Reply:
if it only happens in unit tests, then maybe `mockDynamoQuery` has to be fixed instead

Reply:
Only happens in unit tests

Reply:
then we most likely need to refactor the unit tests

Reply:
Not sure what happened here but this seems to be fixed. I assume someone fixed it and got merged in. Anyways, this is not a problem anymore.